ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log - oav
 
+## 07/05/2023 3.2.11
+
+- Re-enable `additionalProperties` validation where the `liveValidator` configuration has `isArmCall` set to `false`. The version of `oav` that is being used to validate `azure-rest-api-specs` has this behavior enabled, so this is not a breaking change.
```

Comment:
Perhaps I'm missing something but this comment isArmCall=false doesn't seem to match the condition in the code.

Reply:
That's the point. `isArmCall` defaults to `false`. We really want that condition to only activate in the case of the context of an arm call. By changing it to `isArmCall==true` that specialcase only applies when we are actually validating an armRequest. Otherwise, we don't activate the special case behavior anymore.